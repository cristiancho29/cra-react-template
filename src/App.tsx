import { ReactElement } from "react";
import logo from "./logo.svg";

const App = (): ReactElement => {
  return (
    <div className="flex justify-center">
      <header className="flex flex-col items-center">
        <img src={logo} className="w-40 h-40" alt="logo" />
        <p>
          Edit <code>src/App.tsx</code> and save to reload.
        </p>
        <a
          className="App-link"
          href="https://reactjs.org"
          target="_blank"
          rel="noopener noreferrer"
        >
          Learn React
        </a>
      </header>
    </div>
  );
};

export default App;
