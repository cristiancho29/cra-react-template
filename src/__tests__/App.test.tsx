import { render } from "@testing-library/react";
import App from "../App";

describe("App render test", () => {
  it("should render correctly", () => {
    const { container } = render(<App />);
    expect(container).toMatchSnapshot();
  });
});
