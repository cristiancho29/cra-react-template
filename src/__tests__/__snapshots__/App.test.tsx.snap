// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App render test should render correctly 1`] = `
<div>
  <div
    class="flex justify-center"
  >
    <header
      class="flex flex-col items-center"
    >
      <img
        alt="logo"
        class="w-40 h-40"
        src="logo.svg"
      />
      <p>
        Edit 
        <code>
          src/App.tsx
        </code>
         and save to reload.
      </p>
      <a
        class="App-link"
        href="https://reactjs.org"
        rel="noopener noreferrer"
        target="_blank"
      >
        Learn React
      </a>
    </header>
  </div>
</div>
`;
